doc/nrf/ug_nrf9160_gs.rst

```diff
@@ -0,0 +1,502 @@
+
+If your DK  is v1.0.1 or lower, it is preprogrammed with the deprecated nRF9160: Asset Tracker application and you need to update the application firmware to :ref:`asset_tracker_v2`.
+The Asset Tracker v2 simulates sensor data and transmits it to Nordic Semiconductor's cloud solution, `nRF Cloud`_.
+
+Both the Asset Tracker v2 application and nRF Cloud are constantly being improved.
+Therefore, you should update the application and the modem firmware before you start working with the nRF9160 DK.
+
+.. _nrf9160_gs_requirements:
+
+Minimum requirements
+********************
+
+Make sure you have all the required hardware and that your computer has one of the supported operating systems.
+
+Hardware
+========
+
+* nRF9160 DK
+* nano-SIM card that supports :term:`LTE-M` or :term:`Narrowband Internet of Things (NB-IoT)` (the nRF9160 DK contains an iBasis SIM card)
+* micro-USB 2.0 cable
+
+Software
+========
+
+* Windows 8 or Windows 10
+* macOS
+* Linux
+
+.. _nrf9160_gs_installing_software:
+
+Installing the required software
+********************************
+
+To work with the nRF9160 DK firmware and certificates, install `nRF Connect for Desktop`_.
+
+After installing and starting the application, install the following apps:
+
+* Programmer
+* LTE Link Monitor
+
+.. _nrf9160_gs_updating_fw:
+
+Updating the DK firmware
+************************
+
+Download the latest application and modem firmware from the `nRF9160 DK Downloads <nRF9160 product website (compatible downloads)>`_ page and extract to a folder of your choice.
+
+The downloaded zip contains the following firmware:
+
+Application firmware
+  The :file:`img_app_bl` folder contains full firmware images for different applications.
+  When following this guide, use an image for the Asset Tracker v2 application.
+
+  Depending on where you are located, you need the image with either LTE-M or NB-IoT support.
+  See `iBasis IoT network coverage`_ for information about which mode is supported where.
+  If your location has coverage for both, you can choose either one.
+
+Application firmware for Device Firmware Update (DFU)
+  The images in the :file:`img_fota_dfu_bin` and :file:`img_fota_dfu_hex` folders contain firmware images for DFU.
+  When following this guide, you can ignore these images.
+
+Modem firmware
+  The modem firmware is in a zip archive instead of a folder.
+  The zip is named :file:`mfwnrf9160_` followed by the firmware version number.
+  Do not unzip this file.
+
+.. _nrf9160_gs_updating_fw_application:
+
+Updating the application firmware
+=================================
+
+To update the application firmware, complete the following steps:
+
+1. Open the Programmer app.
+#. Make sure the **PROG/DEBUG SW10** switch on the nRF9160 DK is set to **nRF91**.
+   On DK v0.9.0 and earlier, this is the **SW5** switch.
+#. Connect the nRF9160 DK to the computer with a micro-USB cable, and then turn the DK on.
```

Comment:
The original has the step result "In the navigation bar, No devices available changes to Select device." This might be useful information to verify that the device has been detected.

Reply:
This does not happen with the current version of the app. At least for me, the menu reads Select Device at all times.